EssentialFeed/EssentialFeedTests/Feed API/LoadFeedImageCommentsFromRemoteUseCaseTests.swift

```diff
@@ -0,0 +1,225 @@
+//
+//  LoadFeedImageCommentsFromRemoteUseCaseTests.swift
+//  EssentialFeedTests
+//
+//  Created by Ivan Ornes on 9/3/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import XCTest
+import EssentialFeed
+
+class LoadFeedImageCommentsFromRemoteUseCaseTests: XCTestCase {
+	
+	func test_init_doesNotRequestDataFromURL() {
+		let (_, client) = makeSUT()
+		
+		XCTAssertTrue(client.requestedURLs.isEmpty)
+	}
+	
+	func test_loadImageCommentsFromURL_requestsCommentsFromURL() {
+		let image = uniqueImage()
+		let (sut, client) = makeSUT(image: image)
+		let url = URL(string: "https://a-given-url.com/image/\(image.id.uuidString)/comments")!
+		
+		_ = sut.load() { _ in }
+		
+		XCTAssertEqual(client.requestedURLs, [url])
+	}
+	
+	func test_loadImageCommentsFromURLTwice_requestsCommentsFromURLTwice() {
+		let image = uniqueImage()
+		let url = URL(string: "https://a-given-url.com/image/\(image.id.uuidString)/comments")!
+		let (sut, client) = makeSUT(image: image)
```

Comment:
The image and the image id are not needed in this test. You can pass *any* URL to the remote loader.

Remove the image and pass a URL instead:

```suggestion
		let url = URL(string: "https://a-given-url.com")!
		let (sut, client) = makeSUT(url: url)
```